recipes/meta.yaml

```diff
@@ -0,0 +1,56 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "ncview" %}
+{% set version = "2.1.6" %}
+{% set sha256 = "461738db29f9eb942a07e1ba4b57422aa3390d358f853bfd17c42a8c06f13543" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: ftp://cirrus.ucsd.edu/pub/ncview/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - libnetcdf 4.4.*
+    - udunits2
+    - libpng >=1.6.23,<1.7
+
+  run:
+    - libnetcdf 4.4.*
+    - udunits2
+    - libpng >=1.6.23,<1.7
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  commands:
+    - ncview --help
+
+about:
+  home: http://meteora.ucsd.edu/~pierce/ncview_home_page.html
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGLP.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: MIT
+  summary: 'Ncview 2.1.6'
```

Comment:
Do not use the version in the summary. We usually copy that from the website of the original source. In this case:

```
Ncview: a netCDF visual browser
```